### DISCUSSION TITLE
百度网盘无法获取token

### DISCUSSION THREAD

**Discussion by Vermoudol** [AlistGo/alist#3465] [Q&A]
百度网盘点击刷新令牌，无法获取token值，请求协助
![image](https://user-images.githubusercontent.com/104073793/219379132-7e0c3aa6-6efb-4c3f-9fa6-d0d1cca2378d.png)

**Comment by zhuchaojiebao** [comment]
同问

**Reply by yuhao0407** [reply]
同问，什么时候封的，阿里云也不能用了

**Comment by goodrabbit** [comment]
貌似作者更改了API的回调地址导致的。研究了一下，正确获得令牌的链接应该是这个：

https://openapi.baidu.com/oauth/2.0/authorize?response_type=code&client_id=iYCeC9g08h5vuP9UqvPHKKSVrKFXGa1v&redirect_uri=https://alist.nn.ci/tool/baidu/callback&scope=basic,netdisk&qrcode=1

![屏幕截图 2023-02-21 182552](https://user-images.githubusercontent.com/5088561/220320034-4371bd46-a2e4-4c64-92eb-db4f4d381db3.jpg)

**Reply by zhuchaojiebao** [reply]
谢谢您的帮忙
